components/select_team/components/select_team_item.jsx

```diff
@@ -16,7 +16,8 @@ export default class SelectTeamItem extends React.PureComponent {
         loading: PropTypes.bool.isRequired
     };
 
-    handleTeamClick = () => {
+    handleTeamClick = (e) => {
+        e.preventDefault();
```

Comment:
In this case the link will never fire. Is that what we intend? If so, why even use a router `Link`? Just do an `<a onClick={this.handleTeamClick}>`. I think that makes it clearer what's going on at the link site.

Reply:
I think this is what you had originally. But the explanation was wrong, which is why I doubted the fix. Sorry for the run around.

Reply:
No problem. Yes that is what id intended, is to avoid the normal behavior, because you have to "join" to the channel before you be redirected to it. And if there's any problem joining to the channel we show an error in the select_team page.

Reply:
Done! I leave the preventDefault too because I think has sense in this case.

Reply:
Yeah the prevent default is needed now that it is a `a` tag. 👍